network/wsNetwork.go

```diff
@@ -1126,10 +1144,9 @@ func (wn *WebsocketNetwork) ServeHTTP(response http.ResponseWriter, request *htt
 	incomingPeers.Set(float64(wn.numIncomingPeers()), nil)
 }
 
-func (wn *WebsocketNetwork) messageHandlerThread() {
+func (wn *WebsocketNetwork) messageHandlerThread(peersConnectivityCheckCh <-chan time.Time) {
 	defer wn.wg.Done()
-	inactivityCheckTicker := time.NewTicker(connectionActivityMonitorInterval)
-	defer inactivityCheckTicker.Stop()
```

Comment:
Previously each handler would be checked every 3 minutes, now a random handler is checked every 3 minutes?

Is there a downside to having inactive connections remain on the peer list?

Reply:
The downside is that they consume memory, and need to be cleaned out.. eventually.